paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethod.kt

```diff
@@ -414,6 +414,17 @@ internal sealed class SupportedPaymentMethod(
             // AuBecsDebit // TODO: uncomment once we are ready to go live
         )
 
+        /**
+         * This is a list of payment methods that we should tint their icons
+         * when they are selected in UI.
+         */
+        internal fun SupportedPaymentMethod.shouldTintOnSelection(): Boolean {
+            return listOf(
```

Comment:
Use `setOf` for O(1) search.

Reply:
This is what I get for doing work on a friday.